config/src/config/network_config.rs

```diff
@@ -23,21 +23,42 @@ use std::{
     string::ToString,
 };
 
+// TODO: We could possibly move these constants somewhere else, but since they are defaults for the
+//   configurations of the system, we'll leave it here for now.
 /// Current supported protocol negotiation handshake version.
 ///
 /// See [`perform_handshake`] in `network/src/transport.rs`
-// TODO(philiphayes): ideally these constants live somewhere in network/ ...
-// might need to extract into a separate network_constants crate or something.
 pub const HANDSHAKE_VERSION: u8 = 0;
-pub const MAX_FRAME_SIZE: usize = 8 * 1024 * 1024;
+pub const NETWORK_CHANNEL_SIZE: usize = 1024;
+pub const PING_INTERVAL_MS: u64 = 1000;
+pub const PING_TIMEOUT_MS: u64 = 10_000;
+pub const CONNECTIVITY_CHECK_INTERVAL_MS: u64 = 5000;
+pub const PING_FAILURES_TOLERATED: u64 = 10;
```

Comment:
this is really a silly / high number... can we make this 5?

Reply:
Ha, the funny thing is this is what we had before.  Let's do it and make it 5!